node/authorship/src/authorship.rs

```diff
@@ -306,7 +306,7 @@ where
             None,
             Box::pin(async move {
                 // leave some time for evaluation and block finalization (33%)
-                let deadline = (self.now)() + max_duration - max_duration / 3;
+                let deadline = (self.now)() + (max_duration / 3) * 2;
```

Comment:
Isn't it better to multiply first, then divide (not to lose the remainder after integer division)?

```suggestion
                let deadline = (self.now)() + max_duration * 2 / 3;
```

Reply:
I think it's better to divide first in order to avoid `max_duration * 2` overflow

Reply:
Also It's better for deadline to be smaller than to be bigger I think

Reply:
Any reason to do this instead of just ignore clippy on the case?
Any under/overflows aren't possible here and nothing changes if we look at amount of instructions

Reply:
I don't know. I think better to avoid clippy ignore, to have less complexity in code.